src/Test/Functional/Acceptance/AcceptanceCest.php

```diff
@@ -11,7 +11,7 @@
 use Magento\MagentoCloud\Util\ArrayManager;
 
 /**
- * @group php72
+ * This test runs on the latest version of PHP
```

Comment:
Why you don't use something like `@group latest` for easy finding such type of tests?

Reply:
I had a discussion with @shiftedreality and we decided to remove a group for these tests.